src/schml/insert-casts.rkt

```diff
@@ -267,9 +280,64 @@
               ;; Justification: the runtime type is at
               ;; least as precise as t by semantics.
               (MBoxCastedSet! e1 e2 e2-ty))]
-         [else (error 'insert-casts/Mbox-set!T
+
+
+;;        <<<<<<< HEAD
```

Comment:
This is not what I ended up doing to try to fix the problem, but I will try this next time.